src/components/TeacherViews/LessonMedia/LessonMedia.view.tsx

```diff
@@ -38,15 +58,150 @@ type LessonMediaProps =
       lesson: NonCanonicalLesson;
     };
 
-export function LessonMedia(props: LessonMediaProps) {
+export const LessonMedia = (props: LessonMediaProps) => {
   const { isCanonical, lesson } = props;
-  const { lessonTitle, lessonSlug, keyStageTitle } = lesson;
+  const { lessonTitle, lessonSlug, keyStageTitle, mediaClips } = lesson;
 
   const commonPathway = getCommonPathway(
     props.isCanonical ? props.lesson.pathways : [props.lesson],
   );
 
-  const { programmeSlug, unitSlug, subjectTitle } = commonPathway;
+  const { programmeSlug, unitSlug, subjectTitle, yearTitle } = commonPathway;
+
+  const router = useRouter();
+  const { query } = router;
+  const { video } = query;
+
```

Comment:
Maybe we should do this in the query, so that the data comes back in the format we would like, I've had to do something similar on my current ticket